src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -377,10 +377,10 @@ def _get_running_session_mode(
     return session_mode.value[1]
 
 
-def _start_instance(start_instance: bool = None):
+def _start_instance(start_instance: Union[bool, None]):
     """Sets up how to start an instance of fluent."""
-    if not start_instance:
```

Comment:
If `start_instance=False` is passed, the previous code was setting it to True. We should try to evaluate `start_instance` from environment only if it is not passed through `launch_fluent`.

This fixes the issue with `server_info_filename` argument. Following will now work:
```
>>> import ansys.fluent.core as pyfluent
>>> solver = pyfluent.launch_fluent(start_instance=False, server_info_filepath=<server_info_filepath>)
```